### PR TITLE
Origin/feature/enable authentication

### PR DESCRIPTION
**Pull request by manic1990**
Demo changes

### CODE REVIEW

File: BookingSystem/Controllers/BookingController.cs [hunk 1]
```diff
@@ -148,7 +138,50 @@ public async Task<IActionResult> GetAddressByCustomerId(int customerId)
...
+                {
+                    Value = a.RoomID.ToString(),
+                    Text = a.RoomNo.ToString()
+                }).ToList();
+
+                return Json(ListOfRooms);
+            }
+            return Json(string.Empty);
         }
+
```
Comment (Backiaraj07) [inline BookingSystem/Controllers/BookingController.cs:185]: Use try catch block, and log the process

**Review by Backiaraj07 — COMMENTED** [review]
Comments given

File: BookingSystem/Controllers/BookingController.cs (hunk 1 above)
Comment (manic1990) [reply BookingSystem/Controllers/BookingController.cs]: used try catch block and log information.

**Review by Backiaraj07 — APPROVED** [review]
Code reviewed

**Pull request merged**